<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns='http://www.w3.org/1999/xhtml' xml:lang='en' lang='en'>
<head>
<title>obexftp.git - obexftp</title>
<meta name='generator' content='cgit v0.10.1'/>
<meta name='robots' content='index, nofollow'/>
<link rel='stylesheet' type='text/css' href='/cgit-data/cgit.css'/>
<link rel='shortcut icon' href='/favicon.ico'/>
<link rel='alternate' title='Atom feed' href='http://pkgs.fedoraproject.org/cgit/obexftp.git/atom/obexftp-0.24-fix-absurd-install-path.patch?h=master' type='application/atom+xml'/>
</head>
<body>
<div id='cgit'><table id='header'>
<tr>
<td class='logo' rowspan='2'><a href='/cgit/'><img src='/cgit-data/cgit.png' alt='cgit logo'/></a></td>
<td class='main'><a href='/cgit/'>index</a> : <a title='obexftp.git' href='/cgit/obexftp.git/'>obexftp.git</a></td><td class='form'><form method='get' action=''>
<select name='h' onchange='this.form.submit();'>
<option value='f10'>f10</option>
<option value='f11'>f11</option>
<option value='f12'>f12</option>
<option value='f13'>f13</option>
<option value='f14'>f14</option>
<option value='f15'>f15</option>
<option value='f16'>f16</option>
<option value='f17'>f17</option>
<option value='f18'>f18</option>
<option value='f19'>f19</option>
<option value='f20'>f20</option>
<option value='f21'>f21</option>
<option value='f7'>f7</option>
<option value='f8'>f8</option>
<option value='f9'>f9</option>
<option value='fc6'>fc6</option>
<option value='master' selected='selected'>master</option>
</select> <input type='submit' name='' value='switch'/></form></td></tr>
<tr><td class='sub'>obexftp</td><td class='sub right'>Jesse Keating</td></tr></table>
<table class='tabs'><tr><td>
<a href='/cgit/obexftp.git/'>summary</a><a href='/cgit/obexftp.git/refs/'>refs</a><a href='/cgit/obexftp.git/log/obexftp-0.24-fix-absurd-install-path.patch'>log</a><a class='active' href='/cgit/obexftp.git/tree/obexftp-0.24-fix-absurd-install-path.patch'>tree</a><a href='/cgit/obexftp.git/commit/obexftp-0.24-fix-absurd-install-path.patch'>commit</a><a href='/cgit/obexftp.git/diff/obexftp-0.24-fix-absurd-install-path.patch'>diff</a><a href='/cgit/obexftp.git/stats/obexftp-0.24-fix-absurd-install-path.patch'>stats</a></td><td class='form'><form class='right' method='get' action='/cgit/obexftp.git/log/obexftp-0.24-fix-absurd-install-path.patch'>
<select name='qt'>
<option value='grep'>log msg</option>
<option value='author'>author</option>
<option value='committer'>committer</option>
<option value='range'>range</option>
</select>
<input class='txt' type='text' size='10' name='q' value=''/>
<input type='submit' value='search'/>
</form>
</td></tr></table>
<div class='path'>path: <a href='/cgit/obexftp.git/tree/'>root</a>/<a href='/cgit/obexftp.git/tree/obexftp-0.24-fix-absurd-install-path.patch'>obexftp-0.24-fix-absurd-install-path.patch</a></div><div class='content'>blob: 0f5bebdb4a4e9a0faf3743284efedcfc976e25af (<a href='/cgit/obexftp.git/plain/obexftp-0.24-fix-absurd-install-path.patch'>plain</a>)
<table summary='blob content' class='blob'>
<tr><td class='linenumbers'><pre><a id='n1' href='#n1'>1</a>
<a id='n2' href='#n2'>2</a>
<a id='n3' href='#n3'>3</a>
<a id='n4' href='#n4'>4</a>
<a id='n5' href='#n5'>5</a>
<a id='n6' href='#n6'>6</a>
<a id='n7' href='#n7'>7</a>
<a id='n8' href='#n8'>8</a>
<a id='n9' href='#n9'>9</a>
<a id='n10' href='#n10'>10</a>
<a id='n11' href='#n11'>11</a>
<a id='n12' href='#n12'>12</a>
<a id='n13' href='#n13'>13</a>
<a id='n14' href='#n14'>14</a>
<a id='n15' href='#n15'>15</a>
<a id='n16' href='#n16'>16</a>
<a id='n17' href='#n17'>17</a>
<a id='n18' href='#n18'>18</a>
<a id='n19' href='#n19'>19</a>
<a id='n20' href='#n20'>20</a>
</pre></td>
<td class='lines'><pre><code>Index: obexftp-0.24-Source/swig/python/CMakeLists.txt
===================================================================
--- obexftp-0.24-Source.orig/swig/python/CMakeLists.txt
+++ obexftp-0.24-Source/swig/python/CMakeLists.txt
@@ -36,13 +36,13 @@ swig_link_libraries ( obexftp-python
 #
 execute_process (
   COMMAND ${PYTHON_EXECUTABLE} -c
-  	"import site, sys; sys.stdout.write(site.PREFIXES[-1])"
+  	"import site, sys; sys.stdout.write(site.PREFIXES[0])"
   OUTPUT_VARIABLE PYTHON_PREFIX
 )
 file ( TO_CMAKE_PATH "${PYTHON_PREFIX}" PYTHON_PREFIX )
 execute_process (
   COMMAND ${PYTHON_EXECUTABLE} -c
-  	"import site, sys; sys.stdout.write(site.getsitepackages()[-1])"
+  	"import site, sys; sys.stdout.write(site.getsitepackages()[0])"
   OUTPUT_VARIABLE PYTHON_SITE_DIR
 )
 file ( TO_CMAKE_PATH "${PYTHON_SITE_DIR}" PYTHON_SITE_DIR )
</code></pre></td></tr></table>
</div> <!-- class=content -->
<div class='footer'>generated  by cgit v0.10.1 at 2014-12-12 10:03:40 (GMT)</div>
</div> <!-- id=cgit -->
</body>
</html>
